app/views/users/show.html.erb

```diff
@@ -0,0 +1,99 @@
+      <div class="overflow-auto mb-10 md:mb-20">
+        <div class="d-flex flex-row sm:space-y-0">
+
+          <div class="flex flex-col sm:w-1/3 text-center space-y-2 px-6 p1ym4qhs flex flex-col items-center space-y-2 px-6" data-testid="PromoGridItem">
+            <h3 class="profile-card mt-3 md:mt-0 i9prqz1">Events you created
+            </h3>
+          </div>
+
+          <div class="flex flex-col sm:w-1/3 text-center space-y-2 px-6 p1ym4qhs flex flex-col items-center space-y-2 px-6" data-testid="PromoGridItem" style="padding: 0px 10px;">
+            <h3 class="profile-card mt-3 md:mt-0 i9prqz1">Events you will attend
+            </h3>
+          </div>
+
+          <div class="flex flex-col sm:w-1/3 text-center space-y-2 px-6 p1ym4qhs flex flex-col items-center space-y-2 px-6" data-testid="PromoGridItem">
+            <h3 class="profile-card mt-3 md:mt-0 i9prqz1">Past or cancelled events
+            </h3>
+          </div>
+
+        </div>
+      </div>
+  </div>
+<% else %>
+  <% if @user.photo.attached? %>
+    <div class="profile-banner" style="background-image: linear-gradient(rgba(0,0,0,0.3), rgba(0,0,0,0.3)), url('<%= cl_image_path @user.photo.key,crop: :fill %>')">
+      <h2><%= @user.first_name %>, <%= @age %></h2>
+    </div>
+  <% else %>
+    <div class="profile-banner" style="background-image: linear-gradient(rgba(0,0,0,0.3), rgba(0,0,0,0.3)), url('https://images.unsplash.com/photo-1519671282429-b44660ead0a7?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1770&q=80');">
+      <h2><%= @user.first_name %>, <%= @age %></h2>
+    </div>
+  </div>
+  <% end %>
+
+  <div class="container">
+    <div class="about">
+      <div class="about-top d-flex justify-content-between">
+        <h3>About</h3>
+        <p>attended <%= @user.bookings.count %> events</p>
+      </div>
+      <%= @user.about %>
+    </div>
+  </div>
+
+  <div class="container">
+    <button class="invite-btn btn btn-default"> Invite <%= @user.first_name %> to your event.</button>
+  </div>
+
+  <div class="green-background">
+    <div class="container event-box">
+      <h3><%= @user.first_name %> has <%= @user.events.count %> event<%= @user.events.count > 1 || @user.events.count == 0 ? "s": "" %></h3>
+        <% if @user.bookings.nil? %>
+          <p>Stay tuned.</p>
+        <% else %>
+          <% @user.events.each do |event| %>
+            <%= link_to event_path(event) do %>
+              <div class="profile-event-card">
+                <div class="profile-event-card-body">
+                  <div class="event-card-text">
+                    <h5 class="event-card-date"><%= event.date.strftime("%a, %B %Y") %></h5>
+                    <p class="event-card-title"><%= event.title %></em>
+                  </div>
+                  <i class="fas fa-glass-cheers"></i>
+                </div>
+              </div>
+            <% end %>
+          <% end %>
+        <% end %>
+    </div>
+  </div>
+<% end %>
```

Comment:
let's add some <br> somewhere here because of the menubar :-D

Reply:
add some space?

Reply:
ach! '<br>'? :D

Reply:
ach! `<br>`